EA_failure_analysis.py

```diff
@@ -4,9 +4,9 @@
 JSON result files and extracting SQL query details for further investigation.
 """
 
-from typing import Any
+from typing import Any, Dict
```

Comment:
typing.Dict is now being deprecated. From python 3.9, its better to use built-in type hints. So `dict` can directly be used!